.gitignore

```diff
@@ -5,3 +5,4 @@ phpunit.xml
 composer.phar
 package.tar
 /packages.json
+/.phpunit
```

Comment:
wouldn't it be cleaner to just have phpunit in require-dev?
Also easier for first time contributors, now that phpunit on pear, and pear itself, is dead.